search_server.py

```diff
@@ -161,6 +162,11 @@ def do_POST(self):
                     f"   {rich.markup.escape(maybe_content['url'])}"
                     # f"Content: {len(maybe_content['content'])}",
                 )
+
+                # Truncate text 
+                if len(maybe_content) > _TRUNCATE_TEXT_BYTES:
```

Comment:
the `if` is redundant, just leave the next line please (if _TRUNCATE_TEXT_BYTES is too large, it just gets the whole array)

Reply:
Removing the `if` would also make `_TRUNCATE_TEXT_BYTES` being `None` disable the truncation, which would be good